content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+### [](#post-installation-configuration-for-object-store)Post-installation configuration for Object Store
+
+Some post-installation steps must be performed before you can use the HPE Ezmeral Data Fabric Object Store.
+
+You should refer to this document - [Enabling the HPE Ezmeral Data Fabric Object Store](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html) for post-installation configuration.
+
+For the above document, I have a few supplementary notes, which should make your configuration smoother.
+
+#### 1. About the "keytool -noprompt -importcert" command in the docs.
+
+Please do not use this 👇 command in the original document:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias maprca -keystore ${JAVA_HOME}/lib/security/cacerts -storepass <cacerts_truststore>
+```
+
+Instead, use the following command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias maprca -cacerts -storepass <cacerts_truststore>
+```
+
+If you use the Installer to install HPE Ezmeral Data Fabric on a fresh OS, then Installer will automatically install JDK 11, then "-storepass" password is "changeit".
+
+There is another place in the documentation where keytool is used, the command is as follows:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias mosscert -keystore ${JAVA_HOME}/lib/security/cacerts -storepass changeit
+```
+
+You don't need to execute `keytool` twice to import the same cacert certificate file.
+
+I suggest you change "maprca" to something more recognizable.
+
+For example, if you named the cluster "edf-cluster-a.mycompany.com" when you installed the HPE Ezmeral Data Fabric cluster, then you can use the following keytool command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias edf-clustera-ca -keystore ${JAVA_HOME}/lib/security/cacerts - storepass <cacerts_truststore>
+```
+
+The file - /opt/mapr/conf/ca/chain-ca.pem, is a self-signed TLS certificate file created by the Installer when configuring wire-level encryption for the cluster.
+
+Since it's a self-signed TLS certificate, the client (application, or your browser) will not be able to trust the TLS certificate of the server when accessing the HPE Ezmeral Data Fabric server.
+
+This is because the CA certificate used by the self-signed TLS certificate is not publicly trusted.
+
+In any scenario where a self-signed TLS certificate is used, you need to import the self-signed CA certificate into the OS system.
+
+You can use the following command:
+
+```shell
+keytool -list -v -cacerts
+```
+
+to look at things like CA certificates in your JVM.
+
+You'll find something like the following:
```

Comment:
Perhaps modify this sentence to read: "You should now see something like the following:"